lib/ruboty/slack_events/filter/slackify.rb

```diff
@@ -19,7 +19,7 @@ def call(text) #: String
 
         # @rbs text: String
         def replace_user_mentions(text) #: String
-          text.gsub(/@(?<user_name>\w+)/) do |user_mention|
+          text.to_s.gsub(/@(?<user_name>\w+)/) do |user_mention|
```

Comment:
I think it is better to make sure that `text` is String in `Ruboty::Adapters::SlackEvents#say` method
(https://github.com/tomoasleep/ruboty-slack_events/blob/1e07763b0f3080bb5020a33412c3896e959b5bac/lib/ruboty/adapters/slack_events.rb#L34-L55).
All received messages are passed to `Ruboty::Adapters::SlackEvents#say` method. If this method corrects the type of the received messages, other methods does not have to care the type of the received messeges.

Reply:
Thanks for the reply, I'll change it to the way you suggested.